cypress/integration/buy-tshirt.cy.ts

```diff
@@ -34,9 +34,9 @@ describe("Buy a t-shirt", () => {
     shippingPage.acceptTermsService();
     shippingPage.proceedCheckout();
 
-    paymentPage.slectPayMethod();
+    paymentPage.selectPaymentMethod();
     paymentPage.confirmOrder();
 
-    paymentPage.verififyOrder();
+    paymentPage.verifyOrderMessage();
```

Comment:
You can pass the message to verify, in order to not attach the page object to always validate the same message